AirportTicketBooking/InputModels/InputUser/InputUser.cs

```diff
@@ -5,9 +5,30 @@ namespace AirportTicketBooking.InputModels.InputUser;
 
 public static class InputUser
 {
-    public static User InputManager()
+    public static User Input()
     {
-        var manager = new Manager {
+        Console.WriteLine("Enter 1 if you want to create Manager, and 2 if you want to create Passenger");
+        var input = Console.ReadLine();
+        int number;
+        while (string.IsNullOrWhiteSpace(input) || !int.TryParse(input, out number) || (number != 1 && number != 2))
```

Comment:
NIT: you don't use `number` outside of the while loop, so can be just declared on the `TryParse` as a variable

```suggestion
        while (string.IsNullOrWhiteSpace(input) || !int.TryParse(input, out var number) || (number is not 1 or 2))
```